File: radar-auth/src/main/java/org/radarbase/auth/exception/NotAuthorizedException.java
package org.radarbase.auth.exception;

import java.security.GeneralSecurityException;

/**
 * Created by dverbeec on 27/09/2017.
 */
public class NotAuthorizedException extends GeneralSecurityException {
    public NotAuthorizedException() {
        super();
    }

    public NotAuthorizedException(String message) {
        super(message);
    }

    public NotAuthorizedException(Throwable cause) {
        super(cause);
    }

    public NotAuthorizedException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: src/main/java/org/radarbase/management/security/NotAuthorizedException.kt
package org.radarbase.management.security

import java.security.GeneralSecurityException

/**
 * Created by dverbeec on 27/09/2017.
 */
class NotAuthorizedException : GeneralSecurityException {
    constructor() : super()
    constructor(message: String?) : super(message)
    constructor(cause: Throwable?) : super(cause)
    constructor(message: String?, cause: Throwable?) : super(message, cause)
}
